src/drt/src/gc/FlexGC_main.cpp

```diff
@@ -2299,6 +2299,101 @@ void FlexGCWorker::Impl::checkCutSpacing_spc_diff_layer(
   addMarker(std::move(marker));
 }
 
+void FlexGCWorker::Impl::checkLef58CutSpacing_spc_parallelOverlap(
+    gcRect* rect1,
+    gcRect* rect2,
+    frLef58CutSpacingConstraint* con,
+    const gtl::rectangle_data<frCoord>& markerRect)
+{
+  auto net1 = rect1->getNet();
+  auto net2 = rect2->getNet();
+
+  auto prlX = gtl::delta(markerRect, gtl::HORIZONTAL);
+  auto prlY = gtl::delta(markerRect, gtl::VERTICAL);
+  frCoord prl = std::max(prlX, prlY);
+
+  // skip if no parallel overlap
+  if (prl <= 0) {
+    return;
+    // skip if parallel overlap but shares the same above/below metal
```

Comment:
I'm mildly confused which piece of code this comment belongs to.

Reply:
I moved the comment to the correct location